core/src/test/java/org/apache/iceberg/rest/responses/TestCreateNamespaceResponse.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.rest.responses;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.util.Map;
+import org.apache.iceberg.AssertHelpers;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
+import org.apache.iceberg.rest.RequestResponseTestBase;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestCreateNamespaceResponse extends RequestResponseTestBase<CreateNamespaceResponse> {
+
+  /* Values used to fill in response fields */
+  private static final Namespace NAMESPACE = Namespace.of("accounting", "tax");
+  private static final Map<String, String> PROPERTIES = ImmutableMap.of("owner", "Hank");
+  private static final Map<String, String> EMPTY_PROPERTIES = ImmutableMap.of();
+
+  @Test
+  // Test cases that are JSON that can be created via the Builder
+  public void testRoundTripSerDe() throws JsonProcessingException {
+    String fullJson = "{\"namespace\":[\"accounting\",\"tax\"],\"properties\":{\"owner\":\"Hank\"}}";
+    CreateNamespaceResponse req = CreateNamespaceResponse
+        .builder().withNamespace(NAMESPACE).setProperties(PROPERTIES).build();
+    assertRoundTripSerializesEquallyFrom(fullJson, req);
+
+    String jsonEmptyProperties = "{\"namespace\":[\"accounting\",\"tax\"],\"properties\":{}}";
+    CreateNamespaceResponse responseWithExplicitEmptyProperties = CreateNamespaceResponse.builder()
+        .withNamespace(NAMESPACE).setProperties(EMPTY_PROPERTIES).build();
+    assertRoundTripSerializesEquallyFrom(jsonEmptyProperties, responseWithExplicitEmptyProperties);
+
+    CreateNamespaceResponse responseWithImplicitEmptyProperties = CreateNamespaceResponse.builder()
+        .withNamespace(NAMESPACE).build();
+    assertRoundTripSerializesEquallyFrom(jsonEmptyProperties, responseWithImplicitEmptyProperties);
+  }
+
+  @Test
```

Comment:
It looks like there's no equivalent here of the `testCanDeserializeWithoutDefaultValues` for `CreateNamespaceRequest`. Is that different for the response?

Reply:
Added!